software/control/microscope.py

```diff
@@ -179,4 +180,149 @@ def close(self):
         self.camera.close()
         self.microcontroller.close()
         if USE_ZABER_EMISSION_FILTER_WHEEL or USE_OPTOSPIN_EMISSION_FILTER_WHEEL:
-            self.emission_filter_wheel.close()
\ No newline at end of file
+            self.emission_filter_wheel.close()
+
+
+
+class LightSourceType(Enum):
+    Squid = 0
```

Comment:
We should differentiate SquidLED and SquidLaser. Right now they're the same but soon there will be differences.